examples/react-apollo/src/profile/connect-socials.tsx

```diff
@@ -0,0 +1,55 @@
+import { gql } from '@apollo/client'
+import { Card, Group, Title } from '@mantine/core'
+import { useNhostClient, useProviderLink } from '@nhost/react'
+import { useAuthQuery } from '@nhost/react-apollo'
+import { FaGithub } from 'react-icons/fa'
+import AuthLink from 'src/components/AuthLink'
+import { AuthUserProviders } from 'src/generated'
+
+export const ConnectSocials: React.FC = () => {
+  const nhost = useNhostClient()
+  const jwt = nhost.auth.getAccessToken()
+
+  const { github } = useProviderLink({
+    redirectTo: `${window.location.origin}/profile`
+  })
+
+  const AUTH_USER_PROVIDERS = gql`
+    query getAuthUserProviders {
+      authUserProviders {
+        id
+        providerId
+      }
+    }
+  `
+
+  const { data } = useAuthQuery<{
+    authUserProviders: AuthUserProviders[]
+  }>(AUTH_USER_PROVIDERS, {
+    pollInterval: 5000,
+    fetchPolicy: 'cache-and-network'
+  })
+
+  const connectToGithubLink = `${github}&connect=${jwt}`
```

Comment:
shouldn't we have something in the SDK for this? I.e. instead of `nhost.auth.signin({github})` something like `nhost.auth.connect_social({github})`